pkg/minikube/cruntime/cruntime.go

```diff
@@ -163,6 +163,27 @@ type ListImagesOptions struct {
 // ErrContainerRuntimeNotRunning is thrown when container runtime is not running
 var ErrContainerRuntimeNotRunning = errors.New("container runtime is not running")
 
+// ErrRuntimeVersion is the error returned when disk image has incompatible version of service
+type ErrRuntimeVersion struct {
+	Service   string
```

Comment:
does this mean container runtime name ? if yes how about call it "Name"

Reply:
It doesn't. It's any installed service with incompatible version

Reply:
fair enough